x-pack/plugins/security/public/components/management/users/edit_user.js

```diff
@@ -0,0 +1,442 @@
+                            user: {
+                              ...this.state.user,
+                              full_name: event.target.value,
+                            },
+                          });
+                        }}
+                      />
+                    </EuiFormRow>
+                  )}
+                  <EuiFormRow label="Roles">
+                    <EuiComboBox
+                      data-test-subj="userFormRolesDropdown"
+                      placeholder="Add roles"
+                      onChange={this.onRolesChange}
+                      isDisabled={reserved}
+                      name="roles"
+                      options={roles.map(role => {
+                        return { 'data-test-subj': `roleOption-${role.name}`, label: role.name };
+                      })}
+                      selectedOptions={selectedRoles}
+                    />
+                  </EuiFormRow>
+
+                  {isNewUser || showChangePasswordForm ? null : (
+                    <EuiFormRow label="Password">
+                      <EuiLink onClick={this.toggleChangePasswordForm}>Change password</EuiLink>
+                    </EuiFormRow>
+                  )}
+                  {this.changePasswordForm()}
+
+                  <EuiHorizontalRule />
+
+                  {reserved &&
+                    <EuiButton onClick={() => changeUrl(USERS_PATH)}>
+                      Return to user list
+                    </EuiButton>
+                  }
+                  {reserved ? null : (
+                    <EuiFlexGroup responsive={false}>
+                      <EuiFlexItem grow={false}>
+                        <EuiButton
+                          disabled={this.cannotSaveUser()}
+                          fill
+                          data-test-subj="userFormSaveButton"
+                          onClick={() => this.saveUser()}
+                        >
+                          {isNewUser ? 'Create user' : 'Update user'}
+                        </EuiButton>
+                      </EuiFlexItem>
+                      <EuiFlexItem grow={false}>
+                        <EuiButtonEmpty
+                          data-test-subj="userFormCancelButton"
+                          onClick={() => changeUrl(USERS_PATH)}
+                        >
+                          Cancel
+                        </EuiButtonEmpty>
+                      </EuiFlexItem>
+                      <EuiFlexItem grow={true} />
+                      {isNewUser || reserved ? null : (
+                        <EuiFlexItem grow={false}>
+                          <EuiButton
+                            onClick={() => {
+                              this.setState({ showDeleteConfirmation: true });
+                            }}
+                            data-test-subj="userFormDeleteButton"
+                            color="danger"
+                          >
+                            Delete user
```

Comment:
@snide / @cchaos 

Would love some thoughts on forms with 3 action buttons, particularly when the actions are Save/Update, Cancel, and Delete:

1 filled, 1 empty, 1 standard (screenshot from this PR's edit user):
<img width="485" alt="screen shot 2018-07-17 at 4 38 52 pm" src="https://user-images.githubusercontent.com/1965714/42851146-ee42a59e-89df-11e8-9d62-209dd9eba54a.png">

vs

1 filled, 2 empty (screenshot from editing scripted field):
<img width="450" alt="screen shot 2018-07-17 at 4 40 59 pm" src="https://user-images.githubusercontent.com/1965714/42851194-3977105e-89e0-11e8-8f58-0f8df8938a94.png">

Reply:
I'd prefer the later of those two.

Reply:
I think making the delete button an empty button works, but I still like the separation of the delete button to be pushed to the right so it reduces accidental clicks.

Reply:
@bmcconaghy I'll go ahead and fix this one too.

Reply:
@cchaos @nreese merged a PR recently that moves the Delete button in scripted fields to the right, so these two forms will be matching 😄